modules/standard/BigInteger.chpl

```diff
@@ -2706,14 +2661,7 @@ module BigInteger {
     BigInteger.invert(this, a, b);
   }
 
-  // remove
-    /*
-    .. warning::
-
-       bigint.remove is deprecated, use bigint.removeFactor instead
-  */
-  @deprecated
-  ("bigint.remove is deprecated, use bigint.removeFactor instead")
+  @deprecated("bigint.remove is deprecated, use :proc:`bigint.removeFactor` instead")
```

Comment:
For these deprecation messages, how is this printed onto the command line when a user attempts to use it? Will it print the `:proc:` part or will it stay nicely formatted there? (where I was only aware it would look nice in docs, but am no docs/deprecation expert)

Reply:
We have filtering for sphinx markup in deprecation messages, so this is purely a docs change.